knowledge_repo/app/templates/base.html

```diff
@@ -114,7 +114,7 @@
         name: 'knowledge_posts',
         limit: 10,
         display: function (item) {
-          return item;
+          return item.title;
```

Comment:
Can we add author to this? I find it's usually the author/title combo that makes it clear to me what I'm looking at.